content/docs/cucumber/varieties.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Cucumber Varieties
+subtitle: "Varieties and Semantic versioning"
+
+weight: 3
+---
+
+Every [Cucumber implementation](/docs/installation) has its own codebase, release cycle and [semver](https://semver.org/) version that is independent from other Cucumber implementations.
+
+The Cucumber team also maintains a list of *varieties* which defines essential characteristics of various Cucumber versions. A Cucumber release may satisfy zero or more characteristics of a variety. If it satisfies all of them it is a member of that variety.
+
+Marking Cucumber releases as part of a variety serves several purposes:
+
+* Define stretch goals for upcoming Cucumber implementations
```

Comment:
I'd leave this out. The complexity of the code base, deprecation strategy and the feature itself tends to influence when it can be released more so then our mere desire to include it.